test/integration/offchain/healthcheck.test.ts

```diff
@@ -13,7 +13,6 @@ describe('health check', () => {
 
   it('should return 200', async () => {
     const response = await get(app, '/health')
```

Comment:
there needs to be more to the `200` test, asserting that the body contains what we expect. Also a test for if each service is down and what the body look like then. You'll need to mock the services to make them appear 'down'. e.g. https://github.com/digicatapult/dscp-api/blob/639934592c2e5b518ac14079bcccdd490e51f4e1/test/integration/routes.test.js#L97